test/helpers/kubectl.ts

```diff
@@ -155,6 +169,18 @@ export async function waitForServiceAccount(name: string, namespace: string): Pr
   }
 }
 
+export async function waitForCRD(name: string): Promise<void> {
+  // TODO: add some timeout
+  while (true) {
```

Comment:
Can we add it now? Otherwise I feel we will never come back to this 😄 I think we should avoid adding more TODOs unless we really cannot fix it now